### ISSUE TITLE
[Feature] Multi-Tree Setup Management

### ISSUE THREAD

**Issue by Corepex** [gramps-project/gramps-web#427]
It would be nice to manage (create / rename) multiple trees in the UI. 
Letting the administrator assign a tree to a user would also make sense. 

WDYT?

**Comment by DavidMStraub** [comment]
Yes a tree management interface would be useful, however I would rather tend to make that standalone, and not part of the Gramps Web main interface, because site administrators might not even have their own tree (tree ID is not mandatory), and even if they have one, the management of other trees is unrelated to their genealogy.

**Comment by DavidMStraub** [comment]
Right, I was not argueing against a web UI, just argueing against making it part of the SPA; this would be a tool that could also be used by people not even using the Gramps Web UI but only Gramps Web API as backend to something else. So in practice it would just be a separate, lean SPA.

**Comment by Corepex** [comment]
> Right, I was not argueing against a web UI, just argueing against making it part of the SPA; this would be a tool that could also be used by people not even using the Gramps Web UI but only Gramps Web API as backend to something else. So in practice it would just be a separate, lean SPA.

Ah now I get it. 
Maybe i could take care of a "small" SPA (written in React). Would it than be possible to implement a tree switch mechanism that let a user switch between different trees? (im the gramps-web SPA)

I take care of the tree CRUD itself.
